src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,210 @@
+
+Please refer to our [developer resources page](/resources/developer) to learn about our latest package versions.
+
+Custom component styling is discouraged, because the Brightlayer UI’s theme may be suppressed. If the team identifies any styling that should be included as part of the Brightlayer UI component theme, please contact us.
+
+## Read Through Our Website
+
+Please read through our [style guides](/style) and [design pattern guidelines](/patterns). Many examples cited by the webpage came from design audits we did in the past and are representative of common mistakes we observed across multiple projects. Team members may be able to identify low-hanging fruits during this self-audit and address these problems ahead of the audit.
+
+## Address Comments from Previous Audits
+
+If the team has received audit comments in the past, team members should address these audit comments first before requesting a new one. In case audit comments cannot be addressed due to time / technical constraints, the team should inform their auditors about this.
+
+# Request for a Design Audit
+
+Ideally, a design audit is conducted periodically by someone outside the product team. Team members may [request](/community/contactus) a design audit from the Brightlayer UI team.
+
+## Give a Demo
+
+Team members are expected to walk their auditors through the application to help them understand the big picture. This includes all major workflows of the application.
+
+## Grant Auditors Direct Access
+
+Whenever possible, teams are encouraged to create a safe environment for the auditor to test the project directly. This is to help the auditor explore user interactions and different workflow branches.
+
+## Communicate with User Journey
+
+User experience design is centered around users. Instead of talking about the technical nature of your infrastructure, teams should explain to auditors in terms of user journeys. Who will be using the application? What do they care about the most? How will the application solve this user’s problem?
+
+## Communicate the Limit
+
+If the product has been around for a long time, the team should mention to its auditor any legacy issues. Auditors should also be informed of other limitations, such as limited ethernet connection speed, mobile app-exclusive features, etc.
+
+## Define a Proper Scope
+
+One reason we recommend doing design audits periodically is so that product teams do not work alone for too long before they receive any external feedback. This also helps keep the audit scope focused on a few workflows or screens at a time.
+
+If the product is complex or has never been audited, consider arranging only a few key workflows or screens to be audited. A large scope will “dilute” auditors’ attention and tend to contain a lot of repetitive comments. The team shall select the part of the product most important to its users, or that can help teams learn the most from it.
```

Comment:
```suggestion
If the product is complex or has never been audited, consider arranging only a few key workflows or screens to be audited. A large scope will “dilute” auditors’ attention and tend to contain a lot of repetitive comments. The product team should select the part of the product that is either most important to its users, or that can help guide future suggestions for other pages.
```
The second part of the original sentence was difficult for me to understand so I rephrased it arbitrarily.